Dockerfile

```diff
@@ -2,12 +2,17 @@ FROM node:alpine
 
 RUN mkdir www/
 WORKDIR www/
-ADD dist/ .
-COPY package.json .
-RUN yarn install --production
 
-EXPOSE 80
+# Copy and install dependencies
+COPY package.json yarn.lock* ./
+RUN yarn install
+
+# Compile/Run
```

Comment:
Maybe we should be building the application in the build server and just copying the files to the container? So that we don't need to install dev dependencies in the production container.

Reply:
@schalela totally agree, it was actually setup that way initially but Kai wanted it to be super easy for anyone to just run everything all at once just using `docker-compose up -d`.

I figured this would work for now to satisfy that requirement but definitely needs optimization... I'm not a docker expert 😛 so we can put a ticket in the icebox for now to improve this when it becomes an issue. thoughts?

Reply:
Yeah, it's not an actual issue until we have proper CI/CD pipelines. That's why I commented is non-blocking 👍 I'm a big fan of small incremental improvements 😄